api/swagger.yaml

```diff
@@ -6779,6 +6779,9 @@ paths:
             Docker-Experimental:
               type: "boolean"
               description: "If the server is running with experimental mode enabled"
+            Supported-Platforms:
+              type: "string"
+              description: "JSON encoded list of supported platforms"
```

Comment:
Personally I'm not a fan of JSON encoded HTTP headers/query values. I liked the comma separated list. If I'm alone on this I can live with the JSON.

Reply:
I can go either way. I'm not sure we have other examples of passing as JSON encoded, but we certainly use Base64 encoded headers (such as registry auth). I kind of liked the fidelity using a JSON structure, and no need to do string-conversion routines.